must-gather/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM quay.io/openshift/origin-must-gather:4.3.0 as builder
+
+FROM centos:7
```

Comment:
please use a smaller image, e.g. ubi (see https://github.com/openshift-kni/performance-addon-operators/blob/master/openshift-ci/Dockerfile.must-gather#L3)

Reply:
```docker run -it --entrypoint /bin/bash  registry.access.redhat.com/ubi8/ubi-minimal

Reply:
but it has microdnf, will try.

Reply:
done,